src/_cffi_src/openssl/x509v3.py

```diff
@@ -34,6 +34,7 @@
 typedef ... Cryptography_STACK_OF_ASN1_INTEGER;
 typedef ... Cryptography_STACK_OF_GENERAL_SUBTREE;
 typedef ... EXTENDED_KEY_USAGE;
+typedef ... CONF;
```

Comment:
Not, that's just the only place we use it. We could keep conf.py and only have the opaque typedef in it